tests/testthat/error/test-subscript-loc.txt

```diff
@@ -272,3 +272,22 @@ x Can't subset element with unknown name `foo`.
 Error: Must index existing elements.
 x Can't subset element with unknown name `foo`.
 
+
+can optionally extend beyond the end
+====================================
+
+> num_as_location(c(1:5, 7), 3, oob = "extend")
+Error: Can't index beyond the end with non-consecutive locations.
+i The input has size 3.
+x The locations 4 and 7 are not consecutive.
+
+> num_as_location(c(1:5, 7, 1), 3, oob = "extend")
+Error: Can't index beyond the end with non-consecutive locations.
+i The input has size 3.
+x The locations 4 and 7 are not consecutive.
+
+> num_as_location(c(1:5, 7, 1, 10), 3, oob = "extend")
+Error: Can't index beyond the end with non-consecutive locations.
+i The input has size 3.
+x The locations 4, 7 and 10 are not consecutive.
```

Comment:
Should use Oxford comma: The locations 4, 7, and 10 are not consecutive

Reply:
Thanks, I'll fix this outside the PR because this will affect other tests.